.github/workflows/pr-golangci-lint.yaml

```diff
@@ -30,6 +30,8 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@ec5d18412c0aeab7936cb16880d708ba2a64e1ae # tag=v6.2.0
         with:
-          version: v1.60.2
+          version: v1.63.4
```

Comment:
@JoelSpeed Do you maybe want to put this bump into a separate PR? (I think the bump alone already introduced a bunch of findings)

Reply:
Yep, will get that separated out and get that merged in, hopefully that's not too controversial of an update. It auto-fixed all the issues it found

Reply:
All good, thank you!